src/Codeception/Snapshot.php

```diff
@@ -123,6 +129,16 @@ public function shouldRefreshSnapshot($refresh = true)
         $this->refresh = $refresh;
     }
 
+    /**
+     * Show detailed diff if snapshot test fails
+     *
+     * @param bool $showDiff
+     */
+    public function shouldShowDiffOnFail($showDiff = true)
```

Comment:
Could you give an example how to use this method?

If users have to call this method in their code, please document it in https://codeception.com/docs/09-Data#Testing-Dynamic-Data-with-Snapshots page by editing https://github.com/Codeception/codeception.github.com/blob/master/docs/09-Data.md file.

Reply:
As the related issue describes, this is meant to display what is different in the expected VS actual assertion of the snapshot. 

This method can be used by invoking it in the test case or in the snapshot constructor, respectively, eg:
``` php
public function fooTest($I, $snapshot) 
{
    $I->amOnPage('/categories');
    $snapshot->shouldShowDiffOnFail();
    $snapshot->assert();  
}
```

and 

``` php
class Categories extends Snapshot
{
    protected $i;

    public function __construct($I)
    {
        $this->i = $I;
        $this->shouldShowDiffOnFail();
    }

    protected function fetchData()
    {
        return $this->i->grabMultiple('a.category');
    }
}
```

Sure, I will update the readme.